exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -7,205 +7,92 @@ namespace TriviaGame
             {
                 if (isGettingOutOfPenaltyBox)
                 {
-                    Console.WriteLine("Answer was correct!!!!");
-                    purses[currentPlayer]++;
-                    Console.WriteLine(players[currentPlayer]
-                            + " now has "
-                            + purses[currentPlayer]
-                            + " Gold Coins.");
-
-                    bool winner = DidPlayerWin();
-                    currentPlayer++;
-                    if (currentPlayer == players.Count) currentPlayer = 0;
-
-                    return winner;
-                }
-                else
-                {
-                    currentPlayer++;
-                    if (currentPlayer == players.Count) currentPlayer = 0;
-                    return true;
-                }
+                   return PlayerAnsweredCorrect();
 
+                }
 
+                StaticVariables.CurrentPlayer++;
+                if (StaticVariables.CurrentPlayer == _player.players.Count) StaticVariables.CurrentPlayer = 0;
+                return true;
 
             }
-            else
-            {
 
-                Console.WriteLine("Answer was corrent!!!!");
-                purses[currentPlayer]++;
-                Console.WriteLine(players[currentPlayer]
-                        + " now has "
-                        + purses[currentPlayer]
-                        + " Gold Coins.");
+            return PlayerAnsweredCorrect();
+        }
+
+        private bool PlayerAnsweredCorrect()
+        {
+            Console.WriteLine("Answer was correct!!!!");
+            _player.playerScores[StaticVariables.CurrentPlayer]++;
+            Console.WriteLine($"{_player.players[StaticVariables.CurrentPlayer]} now has {_player.playerScores[StaticVariables.CurrentPlayer]} Gold Coins.");
 
-                bool winner = DidPlayerWin();
-                currentPlayer++;
-                if (currentPlayer == players.Count) currentPlayer = 0;
+            bool winner = DidPlayerWin();
+            StaticVariables.CurrentPlayer++;
+            if (StaticVariables.CurrentPlayer == _player.players.Count) StaticVariables.CurrentPlayer = 0;
 
-                return winner;
-            }
+            return winner;
         }
 
         public bool WrongAnswer()
         {
             Console.WriteLine("Question was incorrectly answered");
-            Console.WriteLine(players[currentPlayer] + " was sent to the penalty box");
-            inPenaltyBox[currentPlayer] = true;
+            Console.WriteLine($"{_player.players[StaticVariables.CurrentPlayer]} was sent to the penalty box");
+            _player.inPenaltyBox[StaticVariables.CurrentPlayer] = true;
 
-            currentPlayer++;
-            if (currentPlayer == players.Count) currentPlayer = 0;
+            StaticVariables.CurrentPlayer++;
+            if (StaticVariables.CurrentPlayer == _player.players.Count) StaticVariables.CurrentPlayer = 0;
```

Comment:
Could be good to extract this out into a new method which goes to the next player